server/app.ts

```diff
@@ -40,14 +38,20 @@ const resolvers = {
 
 					return async () => { // Sending function
 						try {
-							statusRet[name] = await plugin.sendMessage(message, verifiedConfig);
+							statusRet.push({
```

Comment:
Instead of pushing you can just return the object, and then when you call `Promise.all` it'll give you an array.